File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/Patterns/Pattern13.java
import java.util.*;
public class Main{
    public static void main(String[] args){
        Scanner scn = new Scanner(System.in);

        //write your code here
        int n=scn.nextInt();
        for(int i=0;i<n;i++){
            int sum=1;
            for(int j=0;j<=i;j++){
                System.out.print(sum+"\t");
                int sumn= sum*(i-j)/(j+1);
                sum=sumn;
            }
            System.out.println();
        }

    }
}

File: AR-VR/devAyushDubey_ayushdubey_2024csit1122_2/Patterns/pattern13.cpp
#include <iostream>
using namespace std;

int fact(int n){
    int sum=n;
    if(n==0)
        return 1;
    else{
        for(int i=n-1;i>1;i--)
            sum = sum*i;
    }
    //cout<<sum;
    return sum;
}

int comb(int n, int k){
    int i;
    int num = fact(n);
    int den = fact(k) * fact((n-k));
    i = num / den;
    //cout<<i;
    return i;
}

int main(){
    int n,i,j;
    cin >> n;
    //cout<<comb(n,0);
    for(i=0;i<n;i++){
        for(j=0;j<=i;j++)
            cout<<comb(i,j)<<"	";
        cout<<endl;
    }
    //write your code here
    
}